src/workflows/workflows.ts

```diff
@@ -77,8 +78,8 @@ export class Workflows {
     this.withdrawalsApi = new WithdrawalsApi(config.api);
   }
 
-  public registerOffchain(signer: Signer, starkWallet: StarkWallet) {
-    return registerOffchainWorkflow(signer, starkWallet, this.usersApi);
+  public registerOffchain(signer: Signer, starkWallet: StarkWallet, options?: RegistrationOptions) {
```

Comment:
Update readme with this parameter please

Reply:
i am not 100% sure if we want to explicitly put this option in the readme, what do you think @maksim-tsvetkov